packages/web-components/src/tab/tab.styles.ts

```diff
@@ -0,0 +1,122 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorCompoundBrandStroke,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorNeutralStroke1Hover,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightSemibold,
+  lineHeightBase300,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host([hidden]) {
+    display: none;
+  }
+  :host {
+    position: relative;
+    display: inline-flex;
+    flex-direction: column;
+    cursor: pointer;
+    box-sizing: border-box;
+    justify-content: center;
+    line-height: ${lineHeightBase300};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    color: ${colorNeutralForeground2};
+    fill: currentcolor;
+    grid-row: 1;
+    padding-top: ${spacingHorizontalM};
+    padding-bottom: ${spacingHorizontalM};
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalMNudge};
+    border-radius: ${borderRadiusMedium};
+  }
+  :host::part(tab-content) {
+    display: inline-flex;
+    flex-direction: column;
+    padding: 0 2px;
+  }
+
+  :host([aria-selected='true']) {
+    color: ${colorNeutralForeground1};
+  }
+
+  :host([aria-selected='true']),
+  :host([aria-selected='true'][disabled]) {
+    font-weight: ${fontWeightSemibold};
+  }
+
+  /* adds hidden textContent to prevent shifting ui on bold / unbolding of text */
+  :host::part(tab-content)::after {
+    content: var(--textContent);
+    visibility: hidden;
+    height: 0;
+    line-height: ${lineHeightBase300};
+    font-weight: ${fontWeightSemibold};
+  }
+
+  :host([aria-selected='true'])::after {
+    background-color: ${colorCompoundBrandStroke};
```

Comment:
Just curious why we are using psaudo elements here? Couldn't this be applied directly to the host?

Reply:
The background color here is for the the active indicator pseudo element. In order to not use a pseudo selector in this case we could add another dom element. Is that preferable to adding a pseudo element and selector?

Reply:
Thats fine just trying to understand whats happening, Thanks!